app/controllers/AccessibilityController.scala

```diff
@@ -93,22 +94,23 @@ class AccessibilityController @Inject() (
   accessibilityProblemConfirmationPage: AccessibilityProblemConfirmationPage,
   errorPage: InternalErrorPage,
   headerRetriever: RefererHeaderRetriever
-)(implicit val appConfig: AppConfig, val executionContext: ExecutionContext)
+)(implicit val appConfig: CFConfig, val executionContext: ExecutionContext)
```

Comment:
I think we should keep these as AppConfig, and add the new config items to the trait

If we think the trait isn't worth it, then just having AppConfig rather than CFConfig makes sense to me - not sure what CFConfig means

Though not sure if that would mean / is worth updates to our tests if we're using the trait for easy passing of config when testing 🤷

Reply:
I was meant to revisit this and completely forgot about it, thanks for the reminder. We can and should definitely continue using AppConfig.